src/Symfony/Component/Validator/CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 CHANGELOG
 =========
 
+4.3.0
```

Comment:
will this still go in 4.3 ?

Reply:
Why not? We have half a year till its release. ;)

Reply:
damn, you are right, 4.2 is the xmas gift 🎁